plugins/obs-webrtc/webrtc/src/ffi/output.rs

```diff
@@ -117,17 +118,7 @@ pub unsafe extern "C" fn obs_webrtc_whip_output_connect(
     };
 
     output.runtime.spawn(async move {
-        let result = output.stream.connect(&url, bearer_token.as_deref()).await;
-        if let Err(e) = result {
-            error!("Failed connecting to whip output: {e:?}");
-            // Close the peer connection so that future writes fail and disconnect the output
-            // TODO: There should be some nuance about a connection failure and a mid-connection failure
-            output
-                .stream
-                .close()
-                .await
-                .unwrap_or_else(|e| error!("Failed closing whip output after error: {e:?}"));
-        }
+        output.stream.connect(&url, bearer_token.as_deref()).await;
```

Comment:
why was the error handling removed?

Reply:
The close was being called twice and the error was already printed in the connect wrapper